apps/modernization-ui/src/apps/patient/add/otherInfoFields/OtherInfoFields.tsx

```diff
@@ -35,6 +35,7 @@ export default function OtherInfoFields({ id, title, coded }: Props) {
                             name="dateOfBirth"
                             render={({ field: { onChange, value, name } }) => (
                                 <DatePickerInput
+                                    toggleButtonAriaLabel="Date of Birth toggle calendar"
```

Comment:
question:  Does this need to be it's own property?  It seems like it can be derived from the label, "Toggle Date of Birth calendar"